remoulade/api/main.py

```diff
@@ -32,6 +32,21 @@ def get_state(message_id):
     return data.asdict()
 
 
+@app.route("/messages/cancel", methods=["POST"])
+def cancel_message():
+    json_data = request.get_json()
+    canceled = False
+    if json_data is not None:
+        try:
+            message_id = json_data["message_id"]
+            backend = remoulade.get_broker().get_cancel_backend()
+            backend.cancel([message_id])
+            canceled = True
+        except Exception:
```

Comment:
It's genearlly a bad practice to catch everything.Let it raise there will be a meaningful error